src/sock.c

```diff
@@ -363,7 +354,7 @@ int getpeer_information (int fd, char *ipaddr, char *string_addr)
         ipaddr[0] = '\0';
         strlcpy (string_addr, "[unknown]", HOSTNAME_LENGTH);
 
-        /* Look up the IP address */
+        /* Look up the IP address */get_gai_error
```

Comment:
this artefact here should cause a build error

Reply:
Sorry, for that.